### PR TITLE
202401 new mendix cloud regions

### PR DESCRIPTION
**Pull request by vaibhav1805**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ The primary hosting locations are as follows:
 * Mendix Cloud Asia Pacific: AWS Singapore
 * Mendix Cloud Asia Pacific: AWS Sydney
 * Mendix Cloud Asia Pacific: AWS Tokyo
+* Mendix Cloud Asia Pacific: AWS Osaka
+* Mendix Cloud Asia Pacific: AWS Seoul
+* Mendix Cloud Asia Pacific: AWS Jakarta
 * Mendix Cloud Canada: AWS Canada
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Should we change this to "Mendix Cloud Canada: AWS Central" to make it more in line with the AWS naming?

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ The primary hosting locations are as follows:
...
 * Mendix Cloud EU: AWS Dublin
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Should we change this to " Mendix Cloud EU: AWS Ireland" to make it more in line with the AWS naming?

Comment (feddovanede) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Same for " Mendix Cloud US East: AWS Northern Virginia", "Mendix Cloud US West: AWS Oregon" and "Mendix Cloud South America: AWS São Paulo"

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md [hunk 3]
```diff
@@ -22,6 +22,9 @@ The primary hosting locations are as follows:
 * Mendix Cloud Asia Pacific: AWS Singapore
 * Mendix Cloud Asia Pacific: AWS Sydney
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Should we change this to "Mendix Cloud AUstralia: AWS Sydney" to make it more in line with the AWS naming?

Comment (feddovanede) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Also, why do we add _AWS_ to all the names?

**Review by feddovanede — COMMENTED** [review]
Add some remarks on naming

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md (hunk 3 above)
Comment (vaibhav1805) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: I'm not sure @feddovanede, maybe if someday we have a multi-cloud setup this convention will offer some more clarity. I can get rid of `AWS` if you want. Should i proceed with that?

Comment (vaibhav1805) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Also for Austrailia as per AWS docs region is Asia Paciic(Sydney), i don't think makes sense to change it to `Mendix Cloud Australia: AWS Sydney`

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md (hunk 1 above)
Comment (vaibhav1805) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Done

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md (hunk 2 above)
Comment (vaibhav1805) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Done

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md (hunk 3 above)
Comment (feddovanede) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: According to this AWS documentation, it is called Australia (Sydney)? https://aws.amazon.com/about-aws/global-infrastructure/regions_az/

Comment (feddovanede) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: I think adding AWS makes sense once we have multi-cloud setup, but as that is not (yet) the case, I don't think adding AWS adds value. I don't think we explicitly mention AWS anywhere in our documentation, except for the Evaluation Guide.

Comment (vaibhav1805) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/_about.md]: Done. I've stripped off the AWS prefix and renamed Syndey region to `Mendix Cloud Australia: Sydney`.

**Review by feddovanede — APPROVED** [review]
Looks good to me!

**Pull request merged**